transport/ws.extension/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If it is a ws extension, shouldn't this be part of ws transport (rather than a separate module) ?